akka-http/src/main/scala/akka/http/javadsl/server/directives/CacheConditionDirectives.scala

```diff
@@ -38,9 +37,9 @@ abstract class CacheConditionDirectives extends BasicDirectives {
    * it on the *outside* of the `withRangeSupport(...)` directive, i.e. `withRangeSupport(...)`
    * must be on a deeper level in your route structure in order to function correctly.
    */
-  @varargs
-  def conditional(lastModified: DateTime, innerRoute: Route, moreInnerRoutes: Route*): Route =
-    RouteStructure.Conditional(lastModified = Some(lastModified))(innerRoute, moreInnerRoutes.toList)
+  //@varargs
+  //def conditional(lastModified: DateTime, innerRoute: Route, moreInnerRoutes: Route*): Route =
+  //  RouteStructure.Conditional(lastModified = Some(lastModified))(innerRoute, moreInnerRoutes.toList)
```

Comment:
There's an explicit TODO item in the top of the PR for these, I promise I won't forget :)